### ISSUE TITLE
Incorrect example in documentation

### ISSUE THREAD

**Issue by ConnectedSystems** [JuliaDynamics/Agents.jl#428]
The example `@agent` macro usage is given [here](https://juliadynamics.github.io/Agents.jl/stable/examples/schelling/) as:

```julia
@agent SchellingAgent GridAgent{2}
    mood::Bool
    group::Int
end
```

But should be:

```julia
@agent SchellingAgent GridAgent{2} begin
    mood::Bool
    group::Int
end
```

**Agents.jl version**

Please provide the version you use (you can do `Pkg.status("Agents")`.

Agents v4.0.4

**Comment by Libbum** [comment]
Indeed. It has been fixed in master, but we should do a 4.0.5 release to patch this omission.

**Comment by Libbum** [comment]
4.0.5 updates the docs.
